hikari/guilds.py

```diff
@@ -262,6 +263,64 @@ class GuildWidget:
     is_enabled: bool = attr.field(repr=True)
     """Whether this embed is enabled."""
 
+    async def fetch_channel(self) -> typing.Optional[channels_.PartialChannel]:
+        """Fetch the widget channel.
+
+        This will be `builtins.None` if not set.
+
+        Returns
+        -------
+        typing.Optional[hikari.channels.PartialChannel]
+            The channel. This will be a _derivative_ of
+            `hikari.channels.PartialChannel`, depending on the type of
+            channel you request for.
+
+            This means that you may get one of
+            `hikari.channels.DMChannel`,
+            `hikari.channels.GroupDMChannel`,
+            `hikari.channels.GuildTextChannel`,
+            `hikari.channels.GuildVoiceChannel`,
+            `hikari.channels.GuildStoreChannel`,
+            `hikari.channels.GuildNewsChannel`.
+
+            Likewise, the `hikari.channels.GuildChannel` can be used to
+            determine if a channel is guild-bound, and
+            `hikari.channels.TextChannel` can be used to determine
+            if the channel provides textual functionality to the application.
```

Comment:
I asked about this a while ago and some said to not change the doc . Anyways will change these ..

Reply:
@ashwinvin Yea, that was my bad, xD. You suggested to make it only text channels, but then I realized it can be any `GuildChannel`, so i forgot to tell you to change it 😓

Reply:
No problem :)